test/hexpm_web/controllers/api/release_controller_test.exs

```diff
@@ -49,6 +49,27 @@ defmodule HexpmWeb.API.ReleaseControllerTest do
       assert log.params["release"]["version"] == "1.0.0"
     end
 
+    test "create release and new package fails with invalid version", %{user: user} do
```

Comment:
could you remove this test? version validation is already tested on a lower level here:

https://github.com/hexpm/hexpm/blob/f9b81e779974050ec80196c8a22bb3e0d331b696/test/hexpm/repository/release_test.exs#L381

Could you replace the controller test for update with a similar lower level test? I think we can similarly extend the existing one.

Reply:
Yup.